README.md

```diff
@@ -276,6 +280,9 @@ The file `common.ts` with type [`AppConfig`](src/config/types.ts) contains impor
 - `tokenLists`: List of token lists including the uri and the parser name to be used to parse the token list. Please update the tokenParserMap in the `src/config/utils.ts` file to include the parser name and the parser function.
 - `sdk`:
   - `cacheTTL`: When the app loads, it will ignore any cached data if it is older than the cacheTTL time in milliseconds. If set to 0, the app will always ignore the cache data and fetch new data on load.
+  - `pairBatchSize`: Amount of pairs batched together in the same multicall when calling strategiesByPair
+  - `blockRangeSize`: Amount of blocks since last call when syncing to latest logs
```

Comment:
Please change this:

```
  - `blockRangeSize`: Amount of blocks since last call when syncing to latest logs
```

into

```
- `blockRangeSize`: Max number of blocks to read logs from in a single call. Each time the SDK reads logs since last processed block it will split the total range into smaller ranges according to this configuration. This should be determined according to the RPC limitations.
```